NOTICE

```diff
@@ -0,0 +1,27 @@
+# THIRD-PARTY LIBRARIES
+
+## MPL-2.0
+
+This project statically links the following [Mozilla Public License 2.0 (MPL-2.0)](https://www.mozilla.org/en-US/MPL/2.0/) libraries.
+The list contains the Go module path and the version of the library used.
+
+- github.com/hashicorp/errwrap-v1.1.0
+- github.com/hashicorp/go-cleanhttp-v0.5.2
+- github.com/hashicorp/go-multierror-v1.1.1
+- github.com/hashicorp/go-retryablehttp-v0.7.7
+- github.com/hashicorp/go-rootcerts-v1.0.2
+- github.com/hashicorp/go-secure-stdlib/strutil-configutil/v0.1.5
+- github.com/hashicorp/hcl-v1.0.1-vault-5
+- github.com/hashicorp/vault-client-go-v0.4.3
+- github.com/letsencrypt/boulder-v0.0.0-20241010192615-6692160cedfa
+
+Modifications to original MPL-2.0 files will be documented and made available.
+
+## LGPL-3.0
+
+This project statically links the following [GNU Lesser General Public License 3.0 (LGPL 3.0)](https://www.gnu.org/licenses/lgpl-3.0.txt) libraries.
+The list contains the Go module path and the version of the library used.
+
+- github.com/juju/fslock-v0.0.0-20160525022230-4d5c94c67b4b
+
```

Comment:
has been fixed already - https://github.com/open-component-model/ocm/pull/1176

Reply:
removed section for TPGL completely now.